.github/workflows/debrepo/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# example build usage:
```

Comment:
While this is not a part of this particular task, it would be still nice to add this as a final step of deployment flow to verify that the deployment for every OS was successful, and package is usable.
Please let me know if you want me to do this piece.

Reply:
As we discussed privately this can be added relatively easily (as the parametrized Docker file is ready) but to avoid complicating and slowing testing we will add this at the end of both developments as a separate PR.

The dockerfile can still be used manually in the meantime.